src/components/GroupNetworkMap/GroupNetworkMap.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { runForceGraph } from './GroupNetworkMapGenerator'
+import styles from './GroupNetworkMap.scss'
+
+export default ({ networkData }) => {
+  const linksData = networkData.links
+  const nodesData = networkData.nodes
+
+  if (linksData.length && nodesData.length) {
+    const containerRef = React.useRef(null)
+
+    React.useEffect(() => {
+      let destroyFn
+
+      if (containerRef.current) {
+        const { destroy } = runForceGraph(containerRef.current, linksData, nodesData)
```

Comment:
seems like this entire function could just be 
`if (containerRef.current) {
return runForceGraph(containerRef.current, linksData, nodesData)
}
return null
`